src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/AldersvilkårVurderingTest.kt

```diff
@@ -0,0 +1,219 @@
+package no.nav.tilleggsstonader.sak.vilkår.vilkårperiode
+
+import no.nav.tilleggsstonader.libs.utils.osloDateNow
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.Feil
+import no.nav.tilleggsstonader.sak.opplysninger.grunnlag.Fødsel
+import no.nav.tilleggsstonader.sak.util.GrunnlagsdataUtil.grunnlagsdataDomain
+import no.nav.tilleggsstonader.sak.util.GrunnlagsdataUtil.lagGrunnlagsdata
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.AldersvilkårVurdering.vurderAldersvilkår
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.VilkårperiodeTestUtil.dummyVilkårperiodeMålgruppe
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.MålgruppeType
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.faktavurderinger.SvarJaNei
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class AldersvilkårVurderingTest {
+    @Test
+    fun `OMSTILLINGSSTØNAD skal gi svar JA`() {
+        val målgruppe = dummyVilkårperiodeMålgruppe()
+        val grunnlagsdata = grunnlagsdataDomain()
+
+        vurderAldersvilkår(målgruppe, grunnlagsdata).also {
+            assertEquals(SvarJaNei.JA, it)
+        }
```

Comment:
Vi bruker `asserj` ellers i kodebasen, her brukes junit sin assertEquals
```suggestion
        assertThat(vurderAldersvilkår(målgruppe, grunnlagsdata)).isEqualTo(SvarJaNei.JA)
```

Reply:
Done: 8b88881597e1114c489492109e42ac85a97098a4